buildpipeline/DotNet-CoreFx-Trusted-Linux.json

```diff
@@ -402,6 +407,9 @@
     "PB_VsoRepositoryName": {
       "value": "DotNet-CoreFX-Trusted"
     },
+    "PB_VsoDockerFileDirUrl": {
+      "value": "https://$(PB_VsoAccountName):$(VsoPassword)@devdiv.visualstudio.com/DevDiv/_git/DotNet-Buildtools-Trusted/src/Microsoft.DotNet.Build.Tasks/PackageFiles/scripts/docker/"
```

Comment:
Sure @weshaggard . I've created https://github.com/dotnet/core-eng/issues/1554 on it.

Could you please take a second look on the updated version?